shared/src/business/useCases/trialSessions/generateTrialCalendarPdfInteractor.test.ts

```diff
@@ -197,6 +197,139 @@ describe('generateTrialCalendarPdfInteractor', () => {
     expect(result.url).toBe(mockPdfUrl.url);
   });
 
+  it('should generate the trial session information pdf using new IrsCalendarAdministratorInfo and return the url to access it', async () => {
+    const irsCalendarAdministratorInfo = {
+      email: 'TEST_EMAIL',
+      name: 'TEST_NAME',
+      phone: 'TEST_PHONE',
+    };
+
+    applicationContext
+      .getPersistenceGateway()
+      .getTrialSessionById.mockReturnValue({
+        ...MOCK_TRIAL_INPERSON,
+        irsCalendarAdministratorInfo,
+      });
+
+    const result = await generateTrialCalendarPdfInteractor(
+      applicationContext,
+      {
+        trialSessionId: MOCK_TRIAL_INPERSON.trialSessionId!,
```

Comment:
small nit (and question): but why would trialSessionId be possibly undefined? Is it not already defined in  `MOCK_TRIAL_INPERSON` object?

Reply:
True, will take a look at that. Outcome from copy and paste tests.